sdk/couchbase-core/src/agent.rs

```diff
@@ -528,13 +530,12 @@ impl Agent {
     fn gen_first_kv_client_configs(
         memd_addrs: &Vec<String>,
         state: &AgentState,
-    ) -> HashMap<String, KvClientConfig> {
+    ) -> Result<HashMap<String, KvClientConfig>> {
```

Comment:
Why does this need to be a Result? It doesn't look like any error case was added?

Reply:
Good spot, a carry-over from when I was trying to fix the FQDN bits.